shaman/controllers/repos/projects.py

```diff
@@ -55,12 +55,17 @@ def index_post(self):
             url=request.json.get("url", ""),
         )
         repo.update_from_json(update_data)
+        archs = request.json.get("archs", [])
```

Comment:
you are always going to get `archs` as a list, even if it is empty, no need to `get()` here (unless you really want it regardless?)